src/js/Hls.js

```diff
@@ -518,6 +528,12 @@ class Hls extends Meister.MediaPlugin {
                 this.recoverFromMediaError();
             }
         }
+
+        if (this.amountOfErrors === this.maxErrors) {
+            // Make sure we are stopping.
+            this.meister.pause();
+            this.meister.error('Too many errors, exceeded maximum', 'HLS-0002', data.details);
```

Comment:
This is not very descriptive. We don't know what kind of error we're tracking so this isn't going to help users or us in fixing the error.